sourcegraph/query_tokens.go

```diff
@@ -175,60 +175,66 @@ func (t *PBToken) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-// Token returns the Token that the PBToken wraps.
-func (t *PBToken) Token() Token {
-	switch {
-	case t.Term != "":
-		return Term(t.Term)
-	case t.AnyToken != "":
-		return AnyToken(t.AnyToken)
-	case t.RepoToken != nil:
-		return *t.RepoToken
-	case t.RevToken != nil:
-		return *t.RevToken
-	case t.FileToken != nil:
-		return *t.FileToken
-	case t.UserToken != nil:
-		return *t.UserToken
-	case t.UnitToken != nil:
-		return *t.UnitToken
-	default:
-		// empty
-		return Term("")
+// GetQueryToken returns the Token that the PBToken wraps.
+func (t *PBToken) GetQueryToken() Token {
+	if term := t.GetTerm(); term != "" {
+		return Term(term)
+	}
+	if anyToken := t.GetAnyToken(); anyToken != "" {
+		return AnyToken(anyToken)
+	}
+	if repoToken := t.GetRepoToken(); repoToken != nil {
+		return *repoToken
+	}
+	if revToken := t.GetRevToken(); revToken != nil {
+		return *revToken
+	}
+	if fileToken := t.GetFileToken(); fileToken != nil {
+		return *fileToken
+	}
+	if userToken := t.GetUserToken(); userToken != nil {
+		return *userToken
+	}
+	if unitToken := t.GetUnitToken(); unitToken != nil {
+		return *unitToken
 	}
+	// empty
+	return Term("")
```

Comment:
Style suggestion, maybe use `if { ... } else if { ... } else if { ... } else { ... }` instead of separate `if`s and a separate last empty case.

Each case returns right away, so at most one will ever execute. The if-else-if-else chain IMO is a better way to express that. It's also more consistent with code elsewhere, and with the previous switch statement (which was removed because each case now needs to call a func first).